### DISCUSSION TITLE
SSVC v2.1

### DISCUSSION THREAD

**Discussion by ahouseholder** [CERTCC/SSVC#285] [General]
The Stakeholder-specific Vulnerability Categorization (SSVC) is a system for prioritizing actions during vulnerability management. SSVC aims to avoid one-size-fits-all solutions in favor of a modular decision-making system with clearly defined and tested parts that vulnerability managers can select and use as appropriate to their context.

Version 2.1 makes the following improvements  on [SSVC version 2.0](https://github.com/CERTCC/SSVC/releases/tag/v2.0):

- Introduced a demo [SSVC Calc App](https://certcc.github.io/SSVC/ssvc-calc/) which became the basis for CISA's [SSVC Calculator](https://www.cisa.gov/ssvc-calculator)
- Updated Deployer tree to use [Automatable](https://github.com/CERTCC/SSVC/blob/main/doc/md_src_files/09_changelog.md#automatable) instead of [Utility](https://github.com/CERTCC/SSVC/blob/main/doc/md_src_files/09_changelog.md#utility), which reduced the size from 108 leaf nodes to 72.
- Adjusted Deployer tree decisions based on stakeholder feedback
- Adjusted Supplier tree decisions based on stakeholder feedback
- Added section on [Sharing Trees With Others](https://github.com/CERTCC/SSVC/blob/main/doc/md_src_files/09_changelog.md#sharing-trees-with-others) including a discussion of decision point scope and decision tree scope.
- Improved clarity of time-sensitivity of some decision points in [Representing Information for Decisions About Vulnerabilities](https://github.com/CERTCC/SSVC/blob/main/doc/md_src_files/09_changelog.md#representing-information-for-decisions-about-vulnerabilities)
- Improved description of [Mission Impact](https://github.com/CERTCC/SSVC/blob/main/doc/md_src_files/09_changelog.md#mission-impact)
- Improved consistency of [Public Safety Impact](https://github.com/CERTCC/SSVC/blob/main/doc/md_src_files/09_changelog.md#public-safety-impact) usage throughout the document and tooling
- Improved consistency of [Human Impact](https://github.com/CERTCC/SSVC/blob/main/doc/md_src_files/09_changelog.md#human-impact) usage throughout the document
- Clarified that known default passwords are an example of [Exploitation](https://github.com/CERTCC/SSVC/blob/main/doc/md_src_files/09_changelog.md#exploitation):PoC
- Clarified that unreachable code (as in unused library features) are [System Exposure](https://github.com/CERTCC/SSVC/blob/main/doc/md_src_files/09_changelog.md#system-exposure):small
- Mention DoD MEF definition in [Mission Impact](https://github.com/CERTCC/SSVC/blob/main/doc/md_src_files/09_changelog.md#mission-impact)
- Updated references to [EPSS](https://www.first.org/epss/) to reflect recent publications
- Refactored markdown files to better track chapter and section numbering, improving findability when editing
- Automated HTML and PDF generation into a [Github Workflow](https://github.com/CERTCC/SSVC/actions/workflows/pandoc_html_pdf.yaml)
- Updated python tools to maintain sync with current SSVC decision models
- Consolidated the SSVC document style guide into a single file in the repository
- Miscellaneous typo fixes and readability improvements (e.g., headings, bulleted lists)

## What's Changed
* Add SSVC v2 PDF to pdfs dir by @ahouseholder in https://github.com/CERTCC/SSVC/pull/145
* fixed typos by @brianadeloye in https://github.com/CERTCC/SSVC/pull/146
* All Schema v2.02 updates. Simplifying the code by @sei-vsarvepalli in https://github.com/CERTCC/SSVC/pull/152
* Somehow missed these schema files from last PR by @sei-vsarvepalli in https://github.com/CERTCC/SSVC/pull/153
* Examples of schema is missing. by @sei-vsarvepalli in https://github.com/CERTCC/SSVC/pull/154
* changed virulence to automatable by @j--- in https://github.com/CERTCC/SSVC/pull/156
* Removing hard-coded final keyword and final outcome by @sei-vsarvepalli in https://github.com/CERTCC/SSVC/pull/157
* recreated CSV files and added a folder for them with readme; updated generation scripts by @j--- in https://github.com/CERTCC/SSVC/pull/161
* Multiple updates 160,163 by @sei-vsarvepalli in https://github.com/CERTCC/SSVC/pull/165
* propagate change in markdown to deployer image; prepare CSVs for sub-trees by @j--- in https://github.com/CERTCC/SSVC/pull/170
* Update CISA-Coordinator-v2.0.3.json by @fruehaufm in https://github.com/CERTCC/SSVC/pull/172
* Update CISA-Coordinator-v2.0.3.json by @fruehaufm in https://github.com/CERTCC/SSVC/pull/173
* Tree updates and code update to fulfill request and recent issues by @sei-vsarvepalli in https://github.com/CERTCC/SSVC/pull/174
* add scripts for coordinator stakeholder. Fix typo in triage graphic by @j--- in https://github.com/CERTCC/SSVC/pull/176
* Update 060_decision-trees.md by @fruehaufm in https://github.com/CERTCC/SSVC/pull/179
* fixed a typo by @fruehaufm in https://github.com/CERTCC/SSVC/pull/182
* Pdf update by @j--- in https://github.com/CERTCC/SSVC/pull/180
* Fixed a typo by @fruehaufm in https://github.com/CERTCC/SSVC/pull/193
* decided on semver scheme for PDF generation script by @j--- in https://github.com/CERTCC/SSVC/pull/194
* Bugfix for Space in values of decision by @sei-vsarvepalli in https://github.com/CERTCC/SSVC/pull/192
* Typo (stray text) in bullet by @j--- in https://github.com/CERTCC/SSVC/pull/196
* Updated the Mission Impact values by @fruehaufm in https://github.com/CERTCC/SSVC/pull/197
* Fixed redundant option for Mission Impact by @fruehaufm in https://github.com/CERTCC/SSVC/pull/187
* Updates to Dryad SSVC Calcultor to use radio buttons in Analyst mode by @sei-vsarvepalli in https://github.com/CERTCC/SSVC/pull/201
* Fix bug in svgzoom by @fneur in https://github.com/CERTCC/SSVC/pull/204
* make the ssvc_v2.py file work with current CSV file names and columns by @ahouseholder in https://github.com/CERTCC/SSVC/pull/207
* fixed a typo by @2shiori17 in https://github.com/CERTCC/SSVC/pull/205
* add github workflow to generate html and pdf artifacts  by @ahouseholder in https://github.com/CERTCC/SSVC/pull/231
* reasona-bly typo by @zmanion in https://github.com/CERTCC/SSVC/pull/232
* Updates to Abbreviated format GH Issue #177 by @sei-vsarvepalli in https://github.com/CERTCC/SSVC/pull/233
* Updating text to conform to Human Impact change by @jeroenh in https://github.com/CERTCC/SSVC/pull/236
* Address time-sensitivity of some decision points by @ahouseholder in https://github.com/CERTCC/SSVC/pull/241
* Add detail about customization, tree sharing, and decision point scope by @ahouseholder in https://github.com/CERTCC/SSVC/pull/242
* Replace Utility with Automatable in Deployer tree by @ahouseholder in https://github.com/CERTCC/SSVC/pull/248
* Two small typo fixes by @jeroenh in https://github.com/CERTCC/SSVC/pull/253
* Improve Mission Impact description by @j--- in https://github.com/CERTCC/SSVC/pull/250
* add subsubsection header for tree versioning by @ahouseholder in https://github.com/CERTCC/SSVC/pull/256
* Remove version strings from file names by @ahouseholder in https://github.com/CERTCC/SSVC/pull/247
* Adjust deployer tree decisions by @ahouseholder in https://github.com/CERTCC/SSVC/pull/262
* Rename markdown files to match current chapter and section names by @ahouseholder in https://github.com/CERTCC/SSVC/pull/263
* mention publicly known default passwords as example of Exploitation:PoC by @ahouseholder in https://github.com/CERTCC/SSVC/pull/265
* Replace κ with *k* to avoid pandoc font errors in build process by @ahouseholder in https://github.com/CERTCC/SSVC/pull/264
* Change default tree to Deployer.json by @ahouseholder in https://github.com/CERTCC/SSVC/pull/258
* Make Public Safety Impact values consistent throughout by @ahouseholder in https://github.com/CERTCC/SSVC/pull/267
* Update analyze_csv.py to reflect csv column name changes by @ahouseholder in https://github.com/CERTCC/SSVC/pull/270
* EPSS changes by @laurie-tyz in https://github.com/CERTCC/SSVC/pull/271
* Update README docs to make finding recent pdf easier by @ahouseholder in https://github.com/CERTCC/SSVC/pull/277
* Adjust Supplier Tree decisions by @ahouseholder in https://github.com/CERTCC/SSVC/pull/276
* Update style guide and acks by @ahouseholder in https://github.com/CERTCC/SSVC/pull/279
* Mention DoD 3020.26 MEF definition in Mission Impact by @cgyarbrough in https://github.com/CERTCC/SSVC/pull/281
* Unreachable code -> System Exposure: Small by @cgyarbrough in https://github.com/CERTCC/SSVC/pull/282
* Update Changelog for v2.1 by @ahouseholder in https://github.com/CERTCC/SSVC/pull/269
* update pdf and html drafts by @ahouseholder in https://github.com/CERTCC/SSVC/pull/283

## New Contributors
* @brianadeloye made their first contribution in https://github.com/CERTCC/SSVC/pull/146
* @fruehaufm made their first contribution in https://github.com/CERTCC/SSVC/pull/172
* @fneur made their first contribution in https://github.com/CERTCC/SSVC/pull/204
* @2shiori17 made their first contribution in https://github.com/CERTCC/SSVC/pull/205
* @zmanion made their first contribution in https://github.com/CERTCC/SSVC/pull/232
* @jeroenh made their first contribution in https://github.com/CERTCC/SSVC/pull/236
* @cgyarbrough made their first contribution in https://github.com/CERTCC/SSVC/pull/281

**Full Changelog**: https://github.com/CERTCC/SSVC/compare/v2.0...v2.1

<hr /><em>This discussion was created from the release <a href='https://github.com/CERTCC/SSVC/releases/tag/v2.1'>SSVC v2.1</a>.</em>

**Comment by j---** [comment]
Excellent work everyone!
Big thanks to @ahouseholder for all your help and leadership here!
